homework/CSharpAdvanced/RepaintingFlowers.cs

```diff
@@ -0,0 +1,20 @@
+﻿
+using System.Collections.Specialized;
+
+namespace CSharpAdvanced
+{
+    public class RepaintingFlowers<T> where T : Flower
+    {
+        public string GetNewColor<T>(T flower, string paint) where T : Flower
```

Comment:
why do you use signature for generic method in non-generic class? I suppose you have already mentioned that T : Flower

Reply:
I suppose in repaint method there should be more functionality. Imagine that you repainted the flower using 1st method, but FORGOT to use 2nd and you client lost money because you sold repainted flowers using default price

Reply:
Also imagine the situation that you get an order to repaint white flower in white. You just sell white rose, but you shouldn't raice price because the ordered color is the initial one.

Reply:
Please refactor you method using pieces of advice above :)